File: objectModel/Python/cdm/persistence/cdmfolder/types/projections/operation_combine_attributes.py
# Copyright (c) Microsoft Corporation. All rights reserved.
# Licensed under the MIT License. See License.txt in the project root for license information.

from typing import List

from cdm.persistence.cdmfolder.types.type_attribute import TypeAttribute
from cdm.persistence.cdmfolder.types.projections.operation_base import OperationBase


class OperationCombineAttributes(OperationBase):
    """OperationCombineAttributes class"""

    def __init__(self):
        super().__init__()

        self.take = None  # type: List[str]
        self.mergeInto = None  # type: TypeAttribute

File: objectModel/CSharp/Microsoft.CommonDataModel.ObjectModel/Persistence/CdmFolder/Types/Projections/OperationCombineAttributes.cs
// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT License. See License.txt in the project root for license information.

namespace Microsoft.CommonDataModel.ObjectModel.Persistence.CdmFolder.Types
{
    using Newtonsoft.Json.Linq;
    using System.Collections.Generic;

    /// <summary>
    /// OperationCombineAttributes class
    /// </summary>
    public class OperationCombineAttributes : OperationBase
    {
        public List<string> Take { get; set; }

        public JToken MergeInto { get; set; }
    }
}
